src/app/components/Input/InputRange.tsx

```diff
@@ -1,58 +1,45 @@
+import { BigNumber } from "@arkecosystem/platform-sdk-support";
 import { useFormField } from "app/components/Form/useFormField";
 import { Range } from "app/components/Range";
-import { useCurrencyDisplay } from "app/hooks";
 import cn from "classnames";
-import React, { useEffect, useMemo, useState } from "react";
+import React, { useMemo } from "react";
 import { getTrackBackground } from "react-range";
 
 import { InputCurrency } from "./InputCurrency";
 import { sanitizeStep } from "./utils";
 
-interface CurrencyInput {
-	display: string;
-	value?: string;
-}
-
 interface Props {
 	disabled?: boolean;
-	value?: CurrencyInput;
-	min: string;
-	max: string;
+	value: string;
+	min: number;
+	max: number;
 	step: number;
 	name?: string;
-	magnitude?: number;
-	onChange?: any;
+	onChange: (value: string) => void;
 }
 
 export const InputRange = React.forwardRef<HTMLInputElement, Props>(
-	({ min, max, step, magnitude, onChange, value, disabled }: Props, ref) => {
-		const { formatRange, convertToCurrency } = useCurrencyDisplay();
+	({ value, onChange, step, disabled, max, ...props }: Props, ref) => {
 		const fieldContext = useFormField();
 
-		const [inputValue, setInputValue] = useState<CurrencyInput>(convertToCurrency(value));
-		const rangeValues = useMemo(() => formatRange(inputValue, max), [formatRange, inputValue, max]);
-
-		const trackBackgroundMinValue = Number(inputValue.display);
-		const minValue = Math.min(Number(min), trackBackgroundMinValue);
+		// @TODO could be simplified after SDK update
+		const rangeValues = useMemo<number[]>(() => {
+			const sanitized = BigNumber.make(value);
+			/* istanbul ignore next */
+			return isNaN(sanitized.toNumber()) ? [] : [Math.min(sanitized.toNumber(), max)];
```

Comment:
```suggestion
			const sanitized = BigNumber.make(value);
			/* istanbul ignore next */
			if (isNaN(sanitized.toNumber()) || sanitized.isZero()) {
				return [];
			}

			return [Math.min(sanitized.toNumber(), max)];
```
@alesmit this should bring back the invisible state for the range, when zero fee is entered